js/modules/k6/http/request_test.go

```diff
@@ -2439,3 +2530,120 @@ func TestBinaryResponseWithStatus0(t *testing.T) {
 	`)
 	require.NoError(t, err)
 }
+
+func GenerateTLSCertificate(t *testing.T, host string, notBefore time.Time, validFor time.Duration) ([]byte, []byte) {
```

Comment:
hmm can you add these helper functions to the `go.k6.io/k6/lib/testutils/httpmultibin` or `go.k6.io/k6/lib/testutils` package, then you won't need to copy them in 2 places almost verbatim?

Reply:
They are different so - no we can't :)

Reply:
Yeah, but I compared them and the differences seemed minor enough to parameterize? In any case, not a deal breaker, it can be fixed later or not at all - anything is better than relying on badssl :sweat_smile: